server/controllers/auth.js

```diff
@@ -1,6 +1,4 @@
 
-const passport = require('passport')
```

Comment:
I don't think you should remove this line because we are using the `isAuthenticated()` method from passport 
WDYT @p-co / @Ekeu  ?

Reply:
In this file, passport module is unused. But it is used in the controller user.js, the controller index.js, and the route passport.js

![image](https://user-images.githubusercontent.com/21025565/105200255-eea83380-5b3f-11eb-9e2b-d3ddf98d734e.png)

Reply:
After some functional tests, there is no dependencies on this variable. req.isAuthenticated is a method accessible when the passport middleware is added in the /routes/index.js